src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -353,6 +358,78 @@ public async Task SendStreamCompletionAsync(string connectionId, string streamId
         await _restClient.SendWithRetryAsync(api, HttpMethod.Post, handleExpectedResponse: null, cancellationToken: cancellationToken);
     }
 
+#if NET7_0_OR_GREATER
+#nullable enable
+    public override async Task<T> InvokeConnectionAsync<T>(string connectionId, string methodName, object?[] args, CancellationToken cancellationToken = default)
+    {
+        // Validate input parameters
+        if (string.IsNullOrEmpty(methodName))
+        {
+            throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(methodName));
+        }
+        if (string.IsNullOrEmpty(connectionId))
+        {
+            throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(connectionId));
+        }
+
+        // Get API endpoint and prepare for the request
+        var api = _restApiProvider.SendClientInvocation(_appName, _hubName, connectionId);
+        InvocationResponse<T>? wrapper = null;
+        string? errorContent = null;
+        bool isSuccess = false;
+        // Send request and capture the response
+        await _restClient.SendMessageWithRetryAsync(
+            api,
+            HttpMethod.Post,
+            methodName,
+            args,
+            async response =>
+            {
+                isSuccess = response.IsSuccessStatusCode;
+
+                if (isSuccess)
+                {
+                    await using var contentStream = await response.Content.ReadAsStreamAsync(cancellationToken);
+
+                    var deserialized = await _restClient.ObjectSerializer.DeserializeAsync(
+                        contentStream,
+                        typeof(InvocationResponse<T>),
+                        cancellationToken);
+
+                    wrapper = deserialized as InvocationResponse<T>
+                        ?? throw new AzureSignalRException("Failed to deserialize response");
+                }
+                else
+                {
+                    errorContent = await response.Content.ReadAsStringAsync(cancellationToken);
+                }
+
+                return isSuccess || response.StatusCode == HttpStatusCode.BadRequest;
+            },
+            cancellationToken);
+
+        // Ensure we have a response
+        if (!isSuccess)
+        {
+            throw new AzureSignalRException(errorContent ?? "Unknown error in response");
+        }
+
+        if (wrapper == null)
+        {
+            throw new AzureSignalRException("Response wrapper is null");
```

Comment:
If (!success) and (wrapper==null), then it should be status code is 404.Does 404 stand for connection not found, or method not found?  Let us provide a clear and helpful error message to the customer.